signer/tuf_on_ci_sign/_signer_repository.py

```diff
@@ -83,7 +84,13 @@ def blue(text: str) -> str:
 
 def _find_changed_roles(known_good_dir: str, signing_event_dir: str) -> list[str]:
     """Return list of roles that exist and have changed in this signing event"""
-    files = glob("*.json", root_dir=signing_event_dir)
+
+    files = []
+    for _, _, filenames in os.walk(signing_event_dir):
+        for filename in filenames:
```

Comment:
Looking at this now, seems like I should have used os.listdir() here for simplicity. Same for the other instance